applications/luci-app-aria2/Makefile

```diff
@@ -1,39 +1,18 @@
 #
-# Copyright (C) 2016 Openwrt.org
+# Copyright (C) 2017-2019 Xingwang Liao <kuoruan@gmail.com>
 #
-# This is free software, licensed under the Apache License, Version 2.0 .
+# This is free software, licensed under the MIT License.
 #
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=luci-app-aria2
-
-# Version == major.minor.patch
-# increase "minor" on new functionality and "patch" on patches/optimization
-PKG_VERSION:=1.0.1
-
-# Release == build
-# increase on changes of translation files
-PKG_RELEASE:=4
-
-PKG_LICENSE:=Apache-2.0
-PKG_MAINTAINER:=Hsing-Wang Liao <kuoruan@gmail.com>
-
-# LuCI specific settings
 LUCI_TITLE:=LuCI Support for Aria2
-LUCI_DEPENDS:=+luci-lib-ipkg +aria2
+LUCI_DEPENDS:=+aria2 +luci-lib-ipkg
 LUCI_PKGARCH:=all
 
-define Package/$(PKG_NAME)/config
-# shown in make menuconfig <Help>
-help
-		$(LUCI_TITLE)
-		.
-		Version: $(PKG_VERSION)-$(PKG_RELEASE)
-		$(PKG_MAINTAINER)
-endef
+PKG_MAINTAINER:=Xingwang Liao <kuoruan@gmail.com>
+PKG_LICENSE:=MIT
```

Comment:
Why did you changed the PKG_LICENSE to MIT?

Reply:
Just because I like it. Most of my open source code is under the MIT license.

Reply:
I don't know if that's so feasible.
What's wrong with the Apache 2.0 license?

@jow- @hnyman Can this be done? From my side I think it is NOK.

Reply:
ping @hnyman any hints from your side

Reply:
No special hints.   Not an license expert. 

There are MIT licensed packages already in the packages repo, so I do not see anything special in that.  

(The new license would naturally only affect the updated new source code versions.)

I woudl say ACK